Sources/DesignResourcesKit/Color.swift

```diff
@@ -32,8 +32,11 @@ public enum DesignSystemColor: String {
     case lines
     case accent
     case icons
+    case iconsSecondary
```

Comment:
Just as a curiosity, how is this secondary colour used? is it for light/dark mode?

Reply:
I think it's for an icon which is on a gray-ish background, like in a button [here](https://www.figma.com/design/N2GbF5HEvopp5iwmAlMwyD/New-Tab-Page-Customization?node-id=656-18939&m=dev).